This crate is impossible to install

I love the idea of this tool and would like to use it, but I just can't compile it. Every time I try (and I've tried several times over many months), I get a torrent of compilation errors.

Contrary to instructions in the readme, it doesn't seem to be compatible with just any Rust nightly. I've even tried using the exact nightly specified in the `rust-toolchain` file, but that seems to be out of sync with the source too:


```
error[E0432]: unresolved import `rustc_middle::ty::BoundRegion`
   --> src/translate.rs:324:31
    |
324 |         use rustc_middle::ty::BoundRegion::BrNamed;
    |                               ^^^^^^^^^^^ `BoundRegion` is a struct, not a module
```


Given how fragile is its compatibility with rustc, maybe you could provide precompiled binary releases instead?

```text
error[E0463]: can't find crate for `rustc_hir`
  --> src/lib.rs:10:1
   |
10 | extern crate rustc_hir;
   | ^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

```

--- a/README.md
+++ b/README.md
@@ -27,12 +27,12 @@ repository and compiled from source or installed from
 of the nighly toolchain is supported at any given time.
 
 
-If you are already using Rust nightly and have successfully installed tools like
-`cargo add` and `cargo clippy`, just do:
+It's recommended to use Rust nightly-2020-11-19. You can install it by using `rustup install nightly-2020-11-19` if you already have rustup.
+Then you can do:
 
 ```sh
-$ rustup component add rustc-dev llvm-tools-preview
-$ cargo +nightly install semverver
+$ rustup component add rustc-dev llvm-tools-preview --toolchain nightly-2020-11-19
+$ cargo +nightly-2020-11-19 install semverver
 ```
 
 You'd also need `cmake` for some dependencies, and a few common libraries (if you hit
@@ -43,7 +43,7 @@ You can also install the newest version of the tool from git:
 
 ```sh
 $ rustup component add rustc-dev llvm-tools-preview --toolchain nightly
-$ cargo +nightly install --git https://github.com/rust-dev-tools/rust-semverver
+$ cargo +nightly-2020-11-19 install --git https://github.com/rust-dev-tools/rust-semverver
 ```
 
 <details>

